store/sqlstore/upgrade.go

```diff
@@ -1211,3 +1213,11 @@ func upgradeDatabaseToVersion537(sqlStore *SqlStore) {
 		saveSchemaVersion(sqlStore, Version5370)
 	}
 }
+
+func upgradeDatabaseToVersion538(sqlStore *SqlStore) {
+	// TODO: uncomment when the time arrive to upgrade the DB for 5.38
```

Comment:
Also, add note to bump `CurrentSchemaVersion`

Reply:
yep, no rush :) thanks!